src/collection.rs

```diff
@@ -323,9 +329,64 @@ impl Collection {
         Ok(())
     }
 
+    /// Drop Partitions
+    pub async fn drop_partition<S: AsRef<str>>(&self, name: S) -> Result<()> {
+        status_to_result(&Some(
+            self.client
+                .clone()
+                .drop_partition(DropPartitionRequest {
+                    base: Some(new_msg(MsgType::DropPartition)),
+                    db_name: "".to_string(),
+                    collection_name: self.schema().name.to_string(),
+                    partition_name: name.as_ref().to_owned(),
+                })
+                .await?
+                .into_inner(),
+        ))
+    }
+
+    /// Load Partitions
+    pub async fn load_partitions<S: ToString, I: IntoIterator<Item = S>>(
```

Comment:
Ref line 131, the method need a `replica_number` parameter

Reply:
fixed it.